README.md

```diff
@@ -1,2 +1,165 @@
-# hello-microverse
-this repository is to leran how to installed linters.
+<a name="readme-top"></a>
+
+<div align="center">
+
+ # AXCELL EVANS
+
+  <h3><b>AXCELL EVANS readme templates</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# Hello Microverse
+This project is a Hello Microverse repository. No complex coding is required for this exercise. The goal is to master all of the tools and best practices learned in previous steps. They will be used in all Microverse projects and most likely in future jobs as well, so it is important to know!.
+1- HTML
+2- css
+3- linters
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">HTML</a></li>
+    <li><a href="https://reactjs.org/">CSS</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+- added index.HTML
+- added Style.CSS
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+# LIVE DEMO 
+
+- [] comming soon
+
+
+- [Live Demo Link](https://google.com)
```

Comment:
- [ ] This link does not link to the live demo of your project. Kindly write "N/A" or coming soon under the header.